src/gateway/types.js

```diff
@@ -280,6 +197,15 @@ export type RenderGatewayOptions = {
      */
     +port: number,
 
+    /**
+     * Optional value in milliseconds for keepalive timeout of the server.
+     * For running in Google Cloud, this should be higher than the load
+     * balancer's own keepalive timeout value, which at time of writing was
+     * 80000ms.
```

Comment:
Might be good to include a link to where that timeout is documented in case we ever need to update it!